error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/ImportSuggestion.java

```diff
@@ -0,0 +1,66 @@
+package tech.picnic.errorprone.bugpatterns;
+
+import static com.google.errorprone.BugPattern.LinkType.CUSTOM;
+import static com.google.errorprone.BugPattern.SeverityLevel.WARNING;
+import static com.google.errorprone.BugPattern.StandardTags.FRAGILE_CODE;
+import static com.google.errorprone.matchers.Matchers.instanceMethod;
+import static tech.picnic.errorprone.bugpatterns.util.Documentation.BUG_PATTERNS_BASE_URL;
+
+import com.google.auto.service.AutoService;
+import com.google.errorprone.BugPattern;
+import com.google.errorprone.VisitorState;
+import com.google.errorprone.bugpatterns.BugChecker;
+import com.google.errorprone.bugpatterns.BugChecker.MethodInvocationTreeMatcher;
+import com.google.errorprone.fixes.SuggestedFix;
+import com.google.errorprone.matchers.Description;
+import com.google.errorprone.matchers.Matcher;
+import com.google.errorprone.util.ASTHelpers;
+import com.sun.source.tree.ExpressionTree;
+import com.sun.source.tree.MethodInvocationTree;
+
+/**
+ * A {@link BugChecker} that flags suggested fixes that involve unconditional imports.
+ *
+ * <p>Such unconditional imports may clash with other imports of the same identifier.
+ */
+@AutoService(BugChecker.class)
+@BugPattern(
+    summary = "Avoid direct invocation of `SuggestedFix#add{,Static}Import`",
+    link = BUG_PATTERNS_BASE_URL + "ImportSuggestion",
+    linkType = CUSTOM,
+    severity = WARNING,
+    tags = FRAGILE_CODE)
+public final class ImportSuggestion extends BugChecker implements MethodInvocationTreeMatcher {
```

Comment:
I'm thinking that `ImportSuggestion` sounds a bit "generic" for its goal. Maybe one doesn't directly link `Suggestion` to the `SuggestedFix` part. Perhaps the word `SuggestedFix` should be in  there as it is specific to that.

Reply:
Hmm, hard to come up with another name. Perhaps `UnqualifiedSuggestedFixImport`?

Reply:
Sounds good IMO, I'll commit the change.